modules/tensor_mechanics/src/materials/crystal_plasticity/CrystalPlasticityKalidindiUpdate.C

```diff
@@ -72,6 +72,12 @@ CrystalPlasticityKalidindiUpdate::initQpStatefulProperties()
   }
 }
 
+void
+CrystalPlasticityKalidindiUpdate::setMaterialVectorSize()
+{
+  CrystalPlasticityStressUpdateBase::setMaterialVectorSize();
+}
+
```

Comment:
No need to override this function in this derived class if the same function is called by the base class. .

Reply:
This one I did to keep the format general for any class that is derived from 'CrystalPlasticityStressUpdateBase' and has non-statefull material property whose size is not known during compile time but being resized during run time e.g, 'CrystalPlasticityTwinningKalidindiUpdate'. The material class 'CrystalPlasticityKalidindiUpdate' is a special case where there is no such resize operation. Let me know if I should remove that override from this one class. @jiangwen84

Reply:
This is a C++ thing. In your case, ```CrystalPlasticityKalidindiUpdate``` will just call the function of ```setMaterialVectorSize``` from its base class. You do not need to add this function in ```CrystalPlasticityKalidindiUpdate``` header and source files.

Reply:
Sure, that makes sense. I will remove that override from `CrystalPlasticityKalidindiUpdate `.